models/group-model.js

```diff
@@ -1,34 +1,37 @@
 const mongoose = require('mongoose');
 
 //Database connection
-mongoose.connect('mongodb://localhost/group', { useNewUrlParser: true })
+mongoose.connect('mongodb://localhost/groups', {
+    useNewUrlParser: true
+  })
   .then(() => console.log('Connected to MongoDB...'))
   .catch(err => console.error('Could not connect to MongoDB...', err));
 
 //Message Object Schema (a sub document of the group message schema)
 const messageObjectSchema = new mongoose.Schema({
```

Comment:
It is a great deal of work put together. Please is it possible as discussed to create an entirely different schema and model of the group message. And tie it by ID to the messageGroup key of the groupSchema?
I strongly suggest you do this so that every schema has a function to perform or less.

groupMessage: {
    type: [messageObjectSchema],
    default: []
  }